contracts/contracts/strategies/balancer/BaseAuraStrategy.sol

```diff
@@ -123,7 +131,7 @@ abstract contract BaseAuraStrategy is BaseBalancerStrategy {
     {
         balancerPoolTokens =
             IERC20(platformAddress).balanceOf(address(this)) +
-            IERC4626(auraRewardPoolAddress).balanceOf(address(this));
+            IERC4626(auraRewardPoolAddress).maxRedeem(address(this));
```

Comment:
good point, also good to know that these things are implementation wise equivalent: 
https://github.com/aurafinance/aura-contracts/blob/45e4507334001a0a8e409be84ced65b73681a480/contracts/compounder/GenericVault.sol#L364-L366